Makefile

```diff
@@ -205,6 +205,29 @@ $(foreach p,$(SUBPROJECTS),$(eval $(call post-build-template,$(p))))
 .PHONY: post-build
 post-build: $(foreach p,$(SUBPROJECTS),post-build-$(p))
 
+###############################################################################
+### dev
+### Starts services in development mode with
+### `ln -s docker-compose.dev.yml docker-compose.override.yml; docker-compose up -d`
+###############################################################################
+define dev-template
+dev-$(1):
+	@if [ -e "$(1)/docker-compose.dev.yml" ]; then \
+	  echo "Starting $(1) in development mode" \
+	  && cd $(1) \
+		&& rm -rf docker-compose.override.yml \
+		&& ln -s docker-compose.dev.yml docker-compose.override.yml \
```

Comment:
Let's remove the `rm` line and use the `ln -sf` option to handle force overwriting existing files.

Reply:
FYI: I was getting following logs if `docker-compose.override.yml` file already existed
![image](https://user-images.githubusercontent.com/22974490/72906118-b832af80-3d57-11ea-8cf1-dd2f12d9bfc2.png)

if that's acceptable then l will remove that line

Reply:
The `-f` option should not print any warnings. At least it doesn't on linux.

Reply:
oh sorry, I was using just `-s`. Thanks, I'll push the changes.